forge-gui/res/cardsfolder/upcoming/slimefoot_and_squee.txt

```diff
@@ -0,0 +1,13 @@
+Name:Slimefoot and Squee
+ManaCost:B R G
+Types:Legendary Creature Fungus Goblin
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, create a 1/1 green Saproling creature token.
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigToken | Secondary$ True | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, create a 1/1 green Saproling creature token.
+SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_saproling
+A:AB$ ChangeZone | Cost$ 1 B R G Sac<1/Saproling> | Origin$ Graveyard | Destination$ Battlefield | ActivationZone$ Graveyard | ValidTgts$ Creature.YouCtrl+Other | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target other creature card in your graveyard | SorcerySpeed$ True | SubAbility$ DBChangeZone | SpellDescription$ Return CARDNAME and up to one other target creature card from your graveyard to the battlefield. Activate only as a sorcery.

```

Comment:
Own instead of Ctrl in Tgts

Reply:
It teaches bad rules practice because these cards shouldn't have a controller at all
Forge is just generous enough to let you abuse it :P

Reply:
Yes